sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 4.3.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 4.3.0-beta.3 (2022-01-11)
 
 ### Other Changes
 
+- Use operation helpers for long-running operations.
```

Comment:
Is this an internal implementation detail, or exposed in the API?

Reply:
Internal, but I had to put something in the changelog or the release scripts would err. I suppose I could just put something vague like "Updates to improve long-term support". @AlexGhiondea?

Reply:
I wouldn't put this as it doesn't impact the customers.

Reply:
I need to include something in the changelog or the scripts complain.

Reply:
Per offline discussion, I'll submit another PR to change this to just "Bug fixes".